src/imagelightbox.js

```diff
@@ -202,13 +202,18 @@
                 });
             },
             captionOn = function () {
-                var description = $(target).find('img').attr('alt');
+                var description;//
+                if ($(target).find('img').length) {
+                    description = $(target).find('img').attr('alt');
+                } else {
+                    description = $(target).data("imagelightbox-caption");
+}
```

Comment:
OK, so one approach is this:
* pull caption from "data-imagelightbox-caption" attribute on link, which is part of the targets set as determined by the presence of the attribute "data-imagelightbox"
* if attribute is null or empty, try creating caption from the alt attribute from the thumbnail image

Or...
* the alt attribute of the thumbnail image could optionally be used for a thumbnail specific caption that could be overlayed.
* and the data-imagelightbox-caption attribute could be used to fill the #imagelightbox-caption element.

third option,
let the user user either option with one taking precedent over the other.